packages/airnode-utilities/package.json

```diff
@@ -12,18 +12,22 @@
     "clean": "rimraf -rf *.tsbuildinfo ./dist ./build *.tgz",
     "compile": "tsc --build tsconfig.json",
     "test": "SILENCE_LOGGER=true jest --selectProjects unit",
+    "test:e2e": "SILENCE_LOGGER=true jest --selectProjects e2e --runInBand",
     "pack": "yarn pack"
   },
   "types": "dist/index",
   "main": "dist/index.js",
   "dependencies": {
+    "@api3/airnode-validator": "^0.7.1",
```

Comment:
I think the latest version is 0.7.2, isn't it?

Reply:
hmm, all other airnode package versions are currently at  0.7.1 as well :thinking:

Reply:
Oh, sorry, it should have been actually `0.7.0`, we're on `master` not on a `v0.7` branch. Never mind then, carry on :smile: